### PR TITLE
Fix issues with new django admin updates

### PR DESCRIPTION
**Pull request by matti-lamppu**
## 🛠️ Changelog
[//]: # "Describe the changes in this pull request here."

- Fixes issues found in testing new Django admin updates added in #1268.
    - Translation fixes
    - Fixes to search functionality
    - Fixes resulting from improper rebasing

## 🧪 Test plan
[//]: # "Help your fellow reviewer and write a short description of the fastest way to test your changes."

- Automated tests

## 🚧 Dependencies
[//]: # "Is this PR dependent on other changes outside this repository? Describe the changes, or add links to them."
[//]: # "e.g. This PR breaks X and is waiting for frontend changes before merging."

- None

## 🎫 Tickets
[//]: # "This pull request resolves all or part of the following ticket(s)."

- [TILA-3039](https://helsinkisolutionoffice.atlassian.net/browse/TILA-3039)
- [TILA-3040](https://helsinkisolutionoffice.atlassian.net/browse/TILA-3040)
- [TILA-3043](https://helsinkisolutionoffice.atlassian.net/browse/TILA-3043)
- [TILA-3425](https://helsinkisolutionoffice.atlassian.net/browse/TILA-3425)

[TILA-3039]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-3039?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[TILA-3040]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-3040?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[TILA-3043]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-3043?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[TILA-3425]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-3425?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: locale/fi/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -2427,66 +2531,107 @@ msgid "7 hours"
...
+
+#: reservation_units/enums.py
+msgctxt "ReservationKind"
+msgid "Direct and season"
+msgstr "Suora ja kausi"
+
+#: reservation_units/enums.py
+msgctxt "PricingType"
+msgid "Paid"
+msgstr "Maksettu"
```
Comment (vergama) [inline locale/fi/LC_MESSAGES/django.po:2551]: Looks like this should more properly be "Maksullinen" as well.

File: locale/fi/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -2929,51 +3074,88 @@ msgstr "Ikäryhmä tälle ryhmälle tai yhdistykselle"
...
+#: reservations/enums.py
+msgctxt "ReservationState"
 msgid "Waiting for payment"
 msgstr "Odottaa maksua"
 
-#: reservations/choices.py
+#: reservations/enums.py
+msgctxt "ReservationState"
 msgid "Confirmed"
 msgstr "Vahvistettu"
```
Comment (vergama) [inline locale/fi/LC_MESSAGES/django.po:3115]: "Hyväksytty"

**Review by vergama — CHANGES_REQUESTED** [review]
Two translation changes, good to go otherwise.

File: locale/fi/LC_MESSAGES/django.po (hunk 2 above)
Comment (matti-lamppu) [reply locale/fi/LC_MESSAGES/django.po]: Fixed

File: locale/fi/LC_MESSAGES/django.po (hunk 1 above)
Comment (matti-lamppu) [reply locale/fi/LC_MESSAGES/django.po]: Fixed

**Review by vergama — APPROVED** [review]

**Pull request merged**